cmd/terramate/cli/cli.go

```diff
@@ -197,53 +197,46 @@ func newCLI(
 		return nil, fmt.Errorf("failed to parse cli args %v: %v", args, err)
 	}
 
-	logLevel := parsedArgs.LogLevel
-	logFmt := parsedArgs.LogFmt
+	configureLogging(parsedArgs.LogLevel, parsedArgs.LogFmt, stderr)
 
-	configureLogging(logLevel, logFmt, stderr)
+	logger := log.With().Logger()
 
-	log.Trace().
-		Str("action", "newCli()").
-		Msg("Get working directory.")
-	wd := parsedArgs.Chdir
-	if wd == "" {
-		wd, err = os.Getwd()
-		if err != nil {
-			return nil, fmt.Errorf("failed to get working directory: %w", err)
-		}
-	}
-
-	logger := log.With().
-		Str("action", "newCli()").
-		Str("stack", wd).
-		Logger()
-
-	logger.Trace().
-		Msgf("Evaluate symbolic links for %q.", wd)
-	wd, err = filepath.EvalSymlinks(wd)
+	cwd, err := os.Getwd()
 	if err != nil {
-		return nil, fmt.Errorf("failed evaluating symlinks for %q: %w", wd, err)
+		log.Fatal().
```

Comment:
Is this supposed to be Fatal? Reading through the conversation further up I got the impression it should be Error not Fatal.

Reply:
if this failed the system is broken anyway... so there is no way around a fatal error here in my opinion... also this is initialization... there should be no defers going on... this should never fail in tests